server/base/src/main/java/org/apache/accumulo/server/conf/codec/PropEncodingV1.java

```diff
@@ -244,11 +244,7 @@ public String print(boolean prettyPrint) {
   }
 
   private void pretty(final boolean prettyPrint, final StringBuilder sb) {
-    if (prettyPrint) {
-      sb.append("\n");
-    } else {
-      sb.append(", ");
-    }
+    prettyPrint ? sb.append("\n") : sb.append(", ");
```

Comment:
After talking with @Manno15 this seems to fix the compiler issue.
```suggestion
    sb.append(prettyPrint ? "\n" : ", ");
```